### ISSUE TITLE
✨ Page - Talent management events

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#12310]
## ✨ Feature

A page to display the talent management events.

## 🎨 Design

@JoshBeveridge 

https://www.figma.com/design/k9lJ77FgLFvpO9L9tVyzID/Talent-nomination-(Employees)?node-id=291-128784&t=35wsjXEj56gvMaGg-4

## 📸 Screenshot

![Image](https://github.com/user-attachments/assets/5b998892-bf9f-42e3-9d9a-d291676ab95c)

## ✅ Acceptance Criteria

- [ ] there is no link to this page in the app yet
- [ ] page exists at URL `/communities/talent-events`
- [ ] `/communities` triggers the 404 pug for now
- [ ] hero, navigation links, sections
- [ ] desktop and mobile layouts
- [ ] cards only displayed if the local time is between the start of the open date and the end of the close date
- [ ] null state well

Out of scope:
- upcoming events
- event history

## 🛑 Blockers

```[tasklist]
### Blocked By
- [ ] #12301
- [ ] #12313
```

**Comment by petertgiles** [comment]
Decision from director review: will change name to "Talent management events"

**Comment by petertgiles** [comment]
> I think this should be out of scope

Oh, good catch.  I'll move that to the AC of 12293.

**Comment by BrookeCameron** [comment]
@mnigh Do you have the URL for this page?

**Comment by petertgiles** [comment]
@BrookeCameron the URL is /communities/talent-events
